### PR TITLE
ED% bug fix

### PR DESCRIPTION
**Pull request by fergushev**
- Moved Reload Config in with the other game settings
- Added Reload Config to in-game settings menu
- Fixed a bug with Enhanced Defense and Enhanced Damage returning 0
- 1H works properly with "hand axe" now
- Added GEMLEVEL (same as GEM) and fixed a bug that prevented unstacked flawless/perfect from filtering properly [GEMLEVEL broke with the else if -> switch rework so fingers crossed that this is the only thing that was missed]

### CODE REVIEW

File: BH/MPQInit.cpp [hunk 1]
```diff
@@ -293,8 +293,8 @@ ItemAttributes ItemAttributeList[] = {
 	{"Chipped Diamond", "gcw", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 65, 0},
 	{"Flawed Diamond", "gfw", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 66, 0},
 	{"Diamond", "gsw", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 68, 0},
-	{"Flawless Diamond", "glws", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 72, 0},
-	{"Perfect Diamond", "gpws", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 80, 0},
+	{"Flawless Diamond", "glw", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 72, 0},
```
Comment (Canite) [inline BH/MPQInit.cpp:296]: Why did you change diamonds to use the unstacked code? It seems like every other flawless/perfect gem in this set is using the stacked code

Comment (fergushev) [reply BH/MPQInit.cpp:296]: The first group (268-297) is the non-stacked ones and it looks like these two were changed to use the stacked codes. Row 308 has the stacked Flawless Diamond, although I'm just now realizing that stacked Perfect Diamond is missing. I'll add it
